packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -162,14 +164,14 @@ class VectorGraphicsCodec {
     // Index Buffer (If non zero)
     // Paint Id.
     buffer._putUint8(_drawVerticesTag);
-    buffer._putInt32(paintId ?? -1);
-    buffer._putInt32(vertices.length);
+    buffer._putUint16(paintId ?? _kMaxUint16);
```

Comment:
Should we assert (here and elsewhere) that we don't every create a real paint with an ID that high?

Reply:
That's a good idea.

Reply:
made `kMaxId` public and documented it, added asserts - do you think asserts are ok or should these be explicit exceptions though?

Reply:
(BTW I'm still using it as a sentinel value for nulls, but assert that no one else tries to use it as an actual value)